ErlangInstaller/MainMenu.swift

```diff
@@ -44,25 +44,25 @@ class MainMenu: NSMenu, NSUserNotificationCenterDelegate, PopoverDelegate {
         }
     }
     
-	func showPopover(_ sender: AnyObject?) {
-		if let button = statusItem!.button {
-			popover.show(relativeTo: button.bounds, of: button, preferredEdge: NSRectEdge.minY)
-			if let controller  = popover.contentViewController as? PopoverViewController {
-			controller.delegate = self
-				_ = Timer.scheduledTimer(timeInterval: 8.0, target: self, selector: #selector(self.closePopover(_:)), userInfo: nil, repeats: false)
-			}
-  		}
-	}
-	
-	func closePopoverFromMainMenu(_ sender: AnyObject?) {
-		self.closePopover(sender)
-	}
-	
-	func closePopover(_ sender: AnyObject?) {
-		popover.performClose(sender)
-	}
- 
-	
+    @objc func showPopover(_ sender: AnyObject?) {
+        if let button = statusItem!.button {
+            popover.show(relativeTo: button.bounds, of: button, preferredEdge: NSRectEdge.minY)
+            if let controller  = popover.contentViewController as? PopoverViewController {
+                controller.delegate = self
+                _ = Timer.scheduledTimer(timeInterval: 8.0, target: self, selector: #selector(self.closePopover(_:)), userInfo: nil, repeats: false)
+            }
+        }
+    }
+    
+    func closePopoverFromMainMenu(_ sender: AnyObject?) {
+        self.closePopover(sender)
+    }
+    
+    @objc func closePopover(_ sender: AnyObject?) {
```

Comment:
Excuse my ignorance, but what does this attribute signify ?

Reply:
The underscore? It means you can call the function without putting a label in the parameter.

So you can do ``closePopover(self)`` instead of ``closePopover(sender: self)``